src/components/common/Input/Input.style.ts

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+import { InputStyledProps } from './Input.type';
+
+export const InputWrapper = styled.div<InputStyledProps>`
```

Comment:
```suggestion
export const InputWrapper = styled(Col)<InputStyledProps>`
```
P5; `globalStyles`에 있는 `Col`을 써도 좋을거 같습니다 !